cluster-autoscaler/cloudprovider/juju/juju_api.go

```diff
@@ -26,7 +37,14 @@ type Manager struct {
 
 func (m *Manager) init() error {
 	var err error
-	m.clients, err = NewClientsUsingClientStore(m.controller, m.model)
+	m.clients, err = NewClientsUsingConnectorConfig(api.SimpleConnectorConfig{
+		ControllerAddress: m.cloudConfig.Endpoint,
```

Comment:
why does this take only one address?  Controllers can be HA?  There should be multiple addresses

Reply:
We discussed this in channel, but ill add a comment here for reference. Currently the simple connector API does not support the ability to have multiple connector addresses listed. However, it seems like [the info struct](https://github.com/juju/juju/blob/fffcf2b1d6cda42866a4811a77e990973f2b2972/api/connector.go#L48) that ultimately consumes the connector config does support that idea. If we can get that worked out, then this PR can move forward.